src/main/java/com/blackducksoftware/integration/fortify/batch/util/RestConnectionHelper.java

```diff
@@ -102,15 +107,17 @@ private static CredentialsRestConnection getRestConnection(final HubServerConfig
 
         CredentialsRestConnection restConnection;
         try {
+            ProxyInfo proxyInfo = null;
```

Comment:
Does any of the hub-common or hub-rest stuff have any proxy handling? Seems like something that should be common.

Reply:
In the latest release of hub-common, the proxy info have been included to credential rest connection. But it didn't have the common mechanism to set the proxy info. Based on your comment, I have created a separate function to set the proxy function. Can you please review it.